src/netstandard/src/netstandard.builds

```diff
@@ -1,26 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What's replacing the builds from this file, now that we've deleted it?

Reply:
We use the inbuilt `TargetFrameworks` multi-dimension resolution in src/netstandard.csproj now.